2024-11-21 12:25:46,287 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Current SDK version is 0.18.6
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Configure stats pid to 44442
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Loading settings from /root/.config/wandb/settings
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Loading settings from /root/Codes/part3/wandb/settings
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'part3/rnn/train.py', 'program_abspath': '/root/Codes/part3/rnn/train.py', 'program': 'rnn/train.py'}
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:_log_setup():533] Logging user logs to /root/Codes/part3/wandb/run-20241121_122546-gzegxnfy/logs/debug.log
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:_log_setup():534] Logging internal logs to /root/Codes/part3/wandb/run-20241121_122546-gzegxnfy/logs/debug-internal.log
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:init():619] calling init triggers
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:init():669] starting backend
2024-11-21 12:25:46,288 INFO    MainThread:44442 [wandb_init.py:init():673] sending inform_init request
2024-11-21 12:25:46,290 INFO    MainThread:44442 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-21 12:25:46,291 INFO    MainThread:44442 [wandb_init.py:init():686] backend started and connected
2024-11-21 12:25:46,297 INFO    MainThread:44442 [wandb_init.py:init():781] updated telemetry
2024-11-21 12:25:46,341 INFO    MainThread:44442 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-21 12:25:47,229 INFO    MainThread:44442 [wandb_init.py:init():867] starting run threads in backend
2024-11-21 12:25:47,352 WARNING MainThread:44442 [wandb_init.py:init():1260] interrupted
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 910, in init
    run._on_start()
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2491, in _on_start
    self._backend.interface.publish_python_packages(working_set())
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 301, in publish_python_packages
    for pkg in working_set:
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/util.py", line 1918, in working_set
    for d in distributions():
  File "/root/miniconda3/lib/python3.8/importlib/metadata.py", line 431, in search
    yield self.joinpath(child)
  File "/root/miniconda3/lib/python3.8/importlib/metadata.py", line 397, in joinpath
    return pathlib.Path(self.root, child)
  File "/root/miniconda3/lib/python3.8/pathlib.py", line 1042, in __new__
    self = cls._from_parts(args, init=False)
  File "/root/miniconda3/lib/python3.8/pathlib.py", line 683, in _from_parts
    drv, root, parts = self._parse_args(args)
KeyboardInterrupt
2024-11-21 12:25:47,355 WARNING MsgRouterThr:44442 [router.py:message_loop():75] message_loop has been closed
